<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchListView</class>
 <widget class="QWidget" name="SearchListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>11</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="filter">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="bookmarks">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAddBookmark">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/main/res/ico/link_add.ico</normaloff>:/new/main/res/ico/link_add.ico</iconset>
   </property>
   <property name="text">
    <string>Add Bookmark</string>
   </property>
   <property name="toolTip">
    <string>Add new URL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRemoveBookmark">
   <property name="text">
    <string>Remove Bookmark</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAddFolder">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/main/res/ico/folder.ico</normaloff>:/new/main/res/ico/folder.ico</iconset>
   </property>
   <property name="text">
    <string>Add Folder</string>
   </property>
   <property name="toolTip">
    <string>Add new folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRemoveFolder">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/main/res/ico/bin.ico</normaloff>:/new/main/res/ico/bin.ico</iconset>
   </property>
   <property name="text">
    <string>Remove Folder</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
